_posts/2017-08-08-CompetitiveCoding.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title: "Competitive Coding: From Novice to Expert"
```

Comment:
the post just clarifies what habits should be avoided by a noob in order to become essentially competitive some day...